package/MDAnalysis/coordinates/XTC.py

```diff
@@ -26,15 +37,15 @@ class XTCWriter(XDRBaseWriter):
     they require significantly less disk space and the loss of precision is
     usually not a problem.
 
-    Parameter
-    ---------
-    filename : str
+    Parameters
+    ----------
+    filename: str
```

Comment:
There should be according to the numpy doc guideline, except that napoleon fails if there is a space before the column.